Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -86,7 +86,7 @@ class Navigator {
      This property can only be modified before creating `Navigator` shared instance, all
      further changes to this property will have no effect. Defaults to `automobile`.
```

Comment:
```suggestion
     further changes to this property will have no effect. Defaults to `automobileAvoidingTraffic`.
```
Probably we should also update default values for `TilesetDescriptorFactory` methods?

Reply:
Makes sense, done